.github/workflows/build.yml

```diff
@@ -1,6 +1,12 @@
 name: Build
 
-on: [push, pull_request]
+on:
+  push:
+    branches-ignore:
+      - master
```

Comment:
if we are skipping tests on the release pipeline then we should run them here for master builds?

Reply:
reverting back to always doing a build.